### PR TITLE
add custom prefix and suffix for cloak by connectclass

### PR DESCRIPTION
**Pull request by Sheogorath-SI**

### CODE REVIEW

File: src/modules/m_cloaking.cpp [hunk 1]
```diff
@@ -231,8 +240,8 @@ class ModuleCloaking : public Module
 			// pfx s1.s2. (xxx.xxx or s3) sfx
 			rv.reserve(prefix.length() + 15 + suffix.length());
 		}
-
-		rv.append(prefix);
+
```
Comment (SaberUK) [inline src/modules/m_cloaking.cpp]: Trailing whitespace should be removed.

**Comment by attilamolnar** [comment]
What's the point in this?

**Comment by SaberUK** [comment]
@attilamolnar It's useful if you have users you want channels to be able to filter e.g. having a connect class that all Tor/BNC users filter into so they can be banned/exempted easily.

**Comment by attilamolnar** [comment]
@SaberUK Put a label on them and ban that using an extban, I don't see why do we need to involve m_cloaking in that.

**Comment by Sheogorath-SI** [comment]
Well, on my net we have users using a meshed vpn for connecting. So we have multiple users from different private IP Ranges. Some users have a full /24 net. so we add some prifix like network/somevpn/ as prefix. so yes, we can also do it with "m_hostchange" but without an real identifier.

**Comment by attilamolnar** [comment]
That's a very special setup. I believe that changing m_cloaking is not the correct approach, rather an extban is a much cleaner approach here.

**Pull request closed without merging**